packer_test/templates/init/mixed_versions.pkr.hcl

```diff
@@ -0,0 +1,9 @@
+packer {
+	required_plugins {
+		tester = {
+			source = "github.com/sylviamoss/comment"
+			version = ">= 0.2.22, <0.2.25" # plugin describe reports 1.0.0 so init must fail
```

Comment:
This one may be problematic, the 0.2.24 release has different versions depending on the OS, not sure how it was built, but as it stands on linux amd64 the version reports 0.2.24 so it succeeds.
We could either make another borked release with a consistent behaviour on all OSes, or we could manually patch the release with the mismatch so the test behaves consistently on all platforms

Reply:
Note: replaced this check with a mondoo/cnspec example instead since they had upstream mismatches with some of their releases